_includes/social.html

```diff
@@ -16,6 +16,7 @@
   {%- if social.youtube_channel -%}<li><a rel="me" href="https://www.youtube.com/channel/{{ social.youtube_channel | cgi_escape | escape }}" title="{{ social.youtube_channel_name | escape | default: 'YouTube' }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#youtube' | relative_url }}"></use></svg></a></li>{%- endif -%}
   {%- if social.telegram -%}<li><a rel="me" href="https://t.me/{{ social.telegram | cgi_escape | escape }}" title="{{ social.telegram | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#telegram' | relative_url }}"></use></svg></a></li>{%- endif -%}
   {%- if social.keybase -%}<li><a rel="me" href="https://keybase.io/{{ social.keybase | cgi_escape | escape }}" title="{{ social.keybase | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#keybase' | relative_url }}"></use></svg></a></li>{%- endif -%}
+  {%- if social.medium -%}<li><a href="https://medium.com/@{{ social.medium | cgi_escape | escape }}" title="{{ social.medium | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#medium' | relative_url }}"></use></svg></a></li>{%- endif -%}
```

Comment:
Consider adding rel="me" for consistency with the others.

Reply:
Gotcha, done.

If anyone comes upon this and wonders `rel="me"` does, [here](https://github.com/jekyll/minima/pull/386) is the original pull request.

Reply:
Great, thanks for sharing.